README.md

```diff
@@ -37,17 +45,17 @@ Operators can behave like managed service providers. Their user interface on the
 
 ## Getting Started 
 
+Check the [Getting Started][olm-getting-started] section.
+
 ### Installation
 
-Install OLM on a Kubernetes or OpenShift cluster by following the [installation guide].
+Install OLM on a Kubernetes cluster by following the [installation guide].
```

Comment:
Hmm, it looks like this is missing a reference? Should this be `... [installation guide][installation guide]`?